templated_output/Arkansas_Nuclear_1.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" standalone="no" ?>
 
-<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
+<simulation>
```

Comment:
These disappear during xml inclusion. I needed a base tag so that no xml fragments occur.
I can change them to any other name but the only things that get added during xml inclusion is the children of tag.